docs/source/api/python/mantid/api/SpectrumInfo.rst

```diff
@@ -7,21 +7,21 @@ This a python binding to the C++ class Mantid::API::SpectrumInfo.
 --------  
 Purpose 
 -------- 
-The purpose of the SpectrumInfo object is to allow the user to access information about the spectra being used in an experiment. The SpectrumInfo object can be used to access information such as the number of spectra, the absolute position of a spectrum as well as the distance from the sample to the source. There are many other methods available as well. 
+The purpose of the ``SpectrumInfo`` object is to allow the user to access information about the spectra being used in an experiment. The ``SpectrumInfo`` object can be used to access information such as the number of spectra, the absolute position of a spectrum as well as the distance from the sample to the source. There are many other methods available as well. 
  
```

Comment:
I think we need to say what a `spectrum` is somewhere in this.  i.e. `A spectrum is a grouping of one or more detectors`